roles/docker/tasks/cleanup.yml

```diff
@@ -1,6 +1,14 @@
 ---
 # This play contains tasks for cleaning up docker
 
+- name: remove all containers
+  shell: unset DOCKER_HOST && docker rm -f -v $(docker ps -a -q)
```

Comment:
Yes, the docker local service is being stopped here and we are trying to clean the containers locally as opposed to targeting any DOCKER_HOST specified in the shell defaults.